src/DssSpell.sol

```diff
@@ -36,40 +55,107 @@ contract DssSpellAction is DssAction {
-        DssExecLib.setIlkAutoLineDebtCeiling("ZRX-A", 10 * MILLION);
-
-        // Increase the YFI-A Maximum Debt Ceiling from 45M to 75M
-        DssExecLib.setIlkAutoLineDebtCeiling("YFI-A", 75 * MILLION);
-
-        // Increase the PSM-USDC-A Debt Ceiling from 1B to 2B
-        DssExecLib.increaseIlkDebtCeiling("PSM-USDC-A", 1 * BILLION, true);
-
-        // Provide Core Unit Budgets
-
-        // Real-World Finance + Governance (Interim Multi-Sig)
-        DssExecLib.sendPaymentFromSurplusBuffer(INTERIM_MULTISIG, (40 + 80) * THOUSAND);
-
-        // Risk
-        DssExecLib.sendPaymentFromSurplusBuffer(RISK, 100_500);
+        bytes32 ilk   = "RWA002-A";
+        uint256 CEIL  = 5 * MILLION;
+        uint256 PRICE = 5_634_804 * WAD;
+        uint256 MAT   = 10_500;
+        uint48 TAU    = 0;
+
+        // https://ipfs.io/ipfs/QmdfuQSLmNFHoxvMjXvv8qbJ2NWprrsvp5L3rGr3JHw18E
+        string memory DOC = "QmdfuQSLmNFHoxvMjXvv8qbJ2NWprrsvp5L3rGr3JHw18E";
+
+        address MIP21_LIQUIDATION_ORACLE =
+            DssExecLib.getChangelogAddress("MIP21_LIQUIDATION_ORACLE");
+
+        address vat = DssExecLib.vat();
+
+        // Sanity checks
+        require(GemJoinAbstract(MCD_JOIN_RWA002_A).vat() == vat, "join-vat-not-match");
+        require(GemJoinAbstract(MCD_JOIN_RWA002_A).ilk() == ilk, "join-ilk-not-match");
+        require(GemJoinAbstract(MCD_JOIN_RWA002_A).gem() == RWA002_GEM, "join-gem-not-match");
+        require(GemJoinAbstract(MCD_JOIN_RWA002_A).dec() == DSTokenAbstract(RWA002_GEM).decimals(), "join-dec-not-match");
+
+        RwaLiquidationLike(MIP21_LIQUIDATION_ORACLE).init(
+            ilk, PRICE, DOC, TAU
+        );
+        (,address pip,,) = RwaLiquidationLike(MIP21_LIQUIDATION_ORACLE).ilks(ilk);
+
+        // Set price feed for RWA002
+        DssExecLib.setContract(DssExecLib.spotter(), ilk, "pip", pip);
+
+        // Init RWA-002 in Vat
+        Initializable(vat).init(ilk);
+        // Init RWA-002 in Jug
+        Initializable(DssExecLib.jug()).init(ilk);
+
+        // Allow RWA-002 Join to modify Vat registry
+        DssExecLib.authorize(vat, MCD_JOIN_RWA002_A);
+
+        // Allow RwaLiquidationOracle to modify Vat registry
+        // DssExecLib.authorize(vat, MIP21_LIQUIDATION_ORACLE);
+
+        // Increase the global debt ceiling by the ilk ceiling
+        DssExecLib.increaseGlobalDebtCeiling(CEIL);
+        // Set the ilk debt ceiling
+        DssExecLib.setIlkDebtCeiling(ilk, CEIL);
+
+        // No dust
+        // DssExecLib.setIlkMinVaultAmount(ilk, 0);
+
+        // stability fee
+        DssExecLib.setIlkStabilityFee(ilk, THREE_PT_FIVE_PCT, false);
+
+        // collateralization ratio
+        DssExecLib.setIlkLiquidationRatio(ilk, MAT);
+
+        // poke the spotter to pull in a price
+        DssExecLib.updateCollateralPrice(ilk);
+
+        // give the urn permissions on the join adapter
+        DssExecLib.authorize(MCD_JOIN_RWA002_A, RWA002_A_URN);
```

Comment:
this was already done here: https://etherscan.io/tx/0xf5efb074b9bf325ce17784b1f975d9b2a69ccfcddb9ce75e450abf84c3d23189

Reply:
Going to comment it out just so we don't loose the context for future spells, as this authorization is needed.